frontend/src/views/Landing/index.tsx

```diff
@@ -55,9 +57,40 @@ const LandingPage = (): JSX.Element => {
   });
   const scrollSection5 = useRef<HTMLDivElement>(null);
 
-  const [cellsHeroNum] = useState("45M");
-  const [datasetsHeroNum] = useState("842");
-  const [cellTypesHeroNum] = useState("682");
+  const { mode, status } = useViewMode();
+  const { data, isLoading, isSuccess } = useFetchDatasets(mode, status);
+
+  const cellsHeroNum: string | null = useMemo(() => {
+    if (!data || isLoading || !isSuccess) return null;
```

Comment:
i think we should hard-code something reasonable in the failure case. maybe just "49.9M"?